crossteam-project-valuation.md

```diff
@@ -0,0 +1,101 @@
+Rather than exactness, you should strive to _**set a worst-case bound**_ on these estimates.
+The idea is for the group of experts to be able to say to each other, "We don't know exactly how long it would take, but we can all agree it's _at least_ this much."
+Specifically, you should estimate a *maximum* reasonable time to consume your project output and *minimum* reasonable times for consumers to otherwise home-roll, use and maintain their own solutions.
+
+> One note about cost-to home-roll.  The cost to home-roll a solution is NOT necessarily (very unlikely, in fact) the same as the cost of making a shared solution.
+Oftentimes for the same functionality the modularity and quality involved in buliding a cross-team, shared solution makes it a noticeably higher investment than a quick, hardcoded implementation used just once.
+
+## Formula
+
+Once you have your worst-case bounds you can value your cross-team project output during a given time frame via the simple formula:
+
+```
+[Time Saved] - [Time Invested]
+
+([Count of New Onboardings] * [Cost to Home-Roll] * [Percent Useful Functionality] + [Count of Usages] * [Maintenance Cost Per Use]) - ([Count of New Onboardings] * [Cost to Onboard])
+
+[Count of New Onboardings] * ([Cost to Home-Roll] * [Percent Useful Functionality] - [Cost to Onboard]) + [Count of Usages] * [Maintenance Cost Per Use]
+```
+
+## Commentary
+
+Despite the trappings of rigor, this process does not yield an exact way to measure cross-team project output.
+In-practice, though, it does give a framework by which you can make a sound decision at how to fund this work.
+After having good, reasonable data according to the above explanation, you should fund dedicated development hours toward running the project up to _**at least**_ of the lesser of the following three levels:
+
+1.  The raw hours saved by the formula above.  Since we're all sure that the formula will produce a number that is below the true number of hours saved, you can have confidence that funding the project up to that point is a sure win for you.
+1.  The amount of time that it takes to support inner sourced contributions to cross-team projects.  Since the contributor would likely have done the work anyway in a one-off fashion, it is worth it to fund the time it takes to faciliate their work going into a shared location.
+1.  Whatever feels good to you.  One intentional side-effect of having a valuation formula is that it  naturally forces measurement of the key points of usage that provide value to consumers.
+Those measurements can be understood and consumed in their raw form to provide you with a gut-feel idea of how valuable is the project.
+
+Some may be concerned about the lack of precision in this  valuation approach.  It's okay for this process to not give  an exact measurement.  It just needs to be accurate enough to accomplish 2 purposes:
```

Comment:
again, I propose to `s/precision/accuracy/`

Reply:
Makes sense.